scripts/lib/utils

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 readonly PROJECTS=(admiral lighthouse shipyard submariner submariner-charts submariner-operator)
+readonly ADMIRAL_CONSUMERS=(lighthouse submariner)
```

Comment:
submariner-operator also consumes admiral  now

Reply:
> submariner-operator also consumes admiral now

Where do you see it? It's not in the go.mod

https://github.com/submariner-io/submariner-operator/blob/master/go.mod

Reply:
ohh wait, I was confusing submariner with submariner-operator.... too much ovn work lately, sorry :D